frontend/src/pages/modelServing/screens/projects/nimUtils.ts

```diff
@@ -77,3 +80,44 @@ export const getNIMServingRuntimeTemplate = async (
     return undefined;
   }
 };
+
+export const updatePVCName = (
+  servingRuntime: ServingRuntimeKind,
+  pvcName: string,
```

Comment:
Perhaps just generate the PVC in here? Do you need it anywhere else?

Reply:
Yes. https://github.com/olavtar/odh-dashboard/blob/pvc-fixes/frontend/src/pages/modelServing/screens/projects/NIMServiceModal/DeployNIMServiceModal.tsx#L240